src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -403,6 +403,142 @@ public async Task CreateAsync(ContentItem contentItem, VersionOptions options)
             }
         }
 
+        public async Task ImportAsync(ContentItem contentItem)
+        {
+            ContentItem existingItem = null;
+            if (!String.IsNullOrEmpty(contentItem.ContentItemVersionId))
+            {
+                existingItem = await GetVersionAsync(contentItem.ContentItemVersionId);
+            }
+
```

Comment:
We could also return if the content item already exist to make sure we don't import these.

```
            if (!String.IsNullOrEmpty(contentItem.ContentItemId))
            {
                if(await GetAsync(contentItem.ContentItemId) != null)
                {
                    return;
                }
            }
```

Reply:
Forget this comment. Actually doing this would return on every items.